cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -247,7 +254,11 @@ def resource_usage(vm_ownership, appliance, provider):
 
     for record in appliance.db.client.session.query(rollups).filter(
             rollups.id.in_(result.subquery())):
-        if record.cpu_usagemhz_rate_average:
+        if record.cpu_usagemhz_rate_average \
```

Comment:
Please wrap these in `()`, that way you don't have to use line-continuation:

```python
if (record.first_check
    or record.second_check
    or record.third_check
    # etc)

Reply:
Done.

Reply:
Ah! I'm sorry @nachandr, lint bot is complaining, my example was a bit wrong. We have to put the binary operator (and, or) _before_ the line break for it to be clean.

```python
if (record.check or
    record.other_check or
    # etc
)